cloud_utils/cache/stores/redis_store.py

```diff
@@ -0,0 +1,41 @@
+import json
+import logging
+from typing import Callable, Tuple
+
+import redis
+
+
+def _cache_key_name(cache_name: str, key: Tuple) -> str:
+    return f"{cache_name}:{json.dumps(key)}"
+
+
+def make_store(
+    redis_client: redis.Redis,
+    name: str,
+    ttl: int = 0,
+) -> Tuple[Callable, Callable]:
+    def get_item(key: Tuple):
+        result = redis_client.get(_cache_key_name(name, key))
```

Comment:
do we sill need this ? iirc make_key gets the name already no ? or maybe make_key should not get a name, and the store is responsible to append it (no that it gets the name)

Reply:
Lets discuss which option is better